README.md

```diff
@@ -1,137 +1,133 @@
-# cosmos-ui
+![the cosmos network](cosmos-github.jpg)
 
-> Human interface to Cosmos. Links to binary downloads coming soon.
+# Cosmos UI
 
-## Warning
+👋 Welcome to the official user interface for the [Cosmos Network](https://cosmos.network/).
 
-This is still alpha-level software as of September 2017. Do not enter in your Cosmos fundraiser seed.
+💻 The Cosmos UI is a desktop application built with [Electron](https://github.com/electron/electron). Cosmos UI runs on macOS 10.9+, Windows 7+, and Debian-based Linux distros.
 
-## Development
+⚠️ This is still alpha-level software. __DO NOT__ enter your Cosmos fundraiser seed into the UI.
 
-To run the dev build, you will need the `gaia` binary installed:
-```golang
+🎉 Releases can be found [here](https://github.com/cosmos/cosmos-ui/releases).
+
+---
+
+### Prerequisites
+
+`gaia` is a prepackaged version of Cosmos SDK and Tendermint. You will need `gaia` installed.
+
+```bash
+# install go, set $GOPATH
+brew install go
+
+# install glide
+brew install glide
+
+# install gaia
 go get github.com/cosmos/gaia
 cd $GOPATH/src/github.com/cosmos/gaia
 git checkout develop
 glide install
 make install
 ```
 
-To run on the default testnet
-```bash
-$ npm run testnet
-```
+You will also need `yarn` to install node modules:
 
-OR, specify a testnet from the github.com/tendermint/testnets repo
 ```bash
-$ npm run testnet <networkName>
-```
+brew install yarn
 
-If you want to see the UI and work on that only, you can use:
-```bash
-$ yarn run uionly
+# in the project folder, run
+yarn
 ```
-This will not start the blockchain conenction.
 
-## Production
+---
 
-You can create production builds to get an app which will connect to an actual blockchain network.
+### Develop
 
-First you will need the prerequisite source repositories installed:
+To run on the UI on the default testnet (`gaia-2-dev`):
+```bash
+$ yarn run testnet
+```
 
+To run the UI on a specific a testnet from the [testnets](https://github.com/tendermint/testnets) repo:
 ```bash
-# install go, set $GOPATH
-brew install go
+$ yarn run testnet <networkName>
+```
 
-# install glide
-brew install glide
+To run the UI with a mocked backend (without connecting to a testnet):
```

Comment:
This should be wrong. uionly will not send or receive anything, not even mocked data.